### PR TITLE
cli.resmgr.download: fix basedir

### PR DESCRIPTION
**Pull request by bertsky**
This likely fixes a bug where (starting from Python 3.8) confusing the argument `basedir` with a (non-existing) keyword arg `basedir=` (and later in the argument list) caused a wrong CWD-based fpath result (IIUC).

### CODE REVIEW

**Comment by bertsky** [comment]
CI failure is completely unrelated: it's caused by extra output
```
/home/circleci/.pyenv/versions/3.7.17/lib/python3.7/site-packages/pydantic/_internal/_config.py:269: UserWarning: Val...have changed in V2:\n* 'schema_extra' has been renamed to 'json_schema_extra'\n  warnings.warn(message, UserWarning)\n
```

interfering with other shell output. 

One of these days I'll get myself a punchbag...

File: ocrd/ocrd/cli/resmgr.py [hunk 1]
```diff
@@ -156,12 +156,12 @@ def download(any_url, no_dynamic, resource_type, path_in_archive, allow_uninstal
                     fpath = resmgr.download(
                         this_executable,
                         resdict['url'],
+                        basedir,
```
Comment (kba) [inline ocrd/ocrd/cli/resmgr.py:159]: Mixing kwargs and args in a long list of arguments is bad obviously.

**Review by kba — APPROVED** [review]

**Comment by kba** [comment]
> CI failure is completely unrelated: it's caused by extra output
> 
> ```
> /home/circleci/.pyenv/versions/3.7.17/lib/python3.7/site-packages/pydantic/_internal/_config.py:269: UserWarning: Val...have changed in V2:\n* 'schema_extra' has been renamed to 'json_schema_extra'\n  warnings.warn(message, UserWarning)\n
> ```
> 
> interfering with other shell output.
> 
> One of these days I'll get myself a punchbag...

Fingers crossed that @joschrew pinning pydantic to `1.*` in #1075 (now merged) makes this go away.

**Comment by kba** [comment]
> Fingers crossed that @joschrew pinning pydantic to `1.*` in #1075 (now merged) makes this go away.

It did

**Pull request merged**